src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM crystallang/crystal:1.1.1
+
+# Add the nodesource ppa to apt.
+RUN wget https://deb.nodesource.com/setup_16.x -O- | bash
+
+# Install nodejs, npm, yarn
+RUN apt-get update
+RUN apt-get install -y nodejs
+RUN npm install -g yarn mix
```

Comment:
Let me know what you think about this: https://stackoverflow.com/a/47451019/497719

I did combine the steps of some sequences because they're unlikely to be used independently. But for these specific examples it seemed like it would hamper development workflows to combine them.

Reply:
The apt update and apt install's definitely need to be combined, otherwise if you cache bust the install layer without busting the update layer, you're gonna have a bad time